projects/pgai/pgai/vectorizer/embedders/openai.py

```diff
@@ -121,24 +111,21 @@ async def embed(self, documents: list[StringDocument]) -> Sequence[EmbeddingVect
         await logger.adebug(f"Chunks produced: {len(documents)}")
         encoder = self._encoder
         context_length = self._context_length
-        is_tokenized = False
-        processed_docs: list[Document] = []
         if encoder is not None and context_length is not None:
+            token_counts: list[int] = []
             # truncate all documents before submitting them to the API
-            is_tokenized = True
-            for document in documents:
+            for i, document in enumerate(documents):
```

Comment:
Would it make sense to rename documents to chunks? considering we have the actual file-document support now I find the terminology a bit confusing.